packages/engine/src/camera/components/FollowCameraComponent.ts

```diff
@@ -29,13 +29,12 @@ export type FollowCameraComponentType = {
   shoulderSide: boolean
   /** Whether the camera auto-rotates toward the target **Default** value is true. */
   locked: boolean
-  /** Camera raycaster */
-  raycaster: Raycaster
   /** Raycast properties */
   raycastProps: RaycastPropsType
 }
 
 export const FollowCameraDefaultValues: FollowCameraComponentType = {
+  targetEntity: 0 as Entity,
```

Comment:
please add a comment note to this specifying entity number 0 will always be localClientEntity

Reply:
0 is the world entity, 1 is the localClientEntity; but this is not guaranteed after the first world, since entities are recycled.